### DISCUSSION TITLE
Add more settings to empire xml file

### DISCUSSION THREAD

**Discussion by githubfan1** [Keriew/augustus#962] [Ideas]
I think that the empire file is a good place for most if not all, the configuration for a map:
Starting conditions:
![starting-conditions](https://github.com/Keriew/augustus/assets/6599012/bdc41b5c-9095-4655-964e-1003b226e52b)

Some of these could be declared with the events already available, the rest may go also in the xml:
![other-conditions](https://github.com/Keriew/augustus/assets/6599012/e9548a0b-4fca-47d0-9922-cd2450ebfa24)
 any idea?

**Comment by PaxBotanica** [comment]
I would only add multiple enemies, Enemy 1,2,3 etc with the timeline invasions defined for each.
